src/ckb/channel.rs

```diff
@@ -504,10 +509,6 @@ impl<S> ChannelActor<S> {
         match flags {
             CommitmentSignedFlags::SigningCommitment(flags) => {
                 let flags = flags | SigningCommitmentFlags::OUR_COMMITMENT_SIGNED_SENT;
-                // Normally commitment number will be incremented after received a RevokeAndAck message.
-                // But here channel has not been etablished yet, so we will not receive RevokeAndAck message.
-                // We increment the commitment number here instead.
-                state.increment_local_commitment_number();
```

Comment:
Could you explain why this is no longer needed?

Reply:
This commit has relevant information about this change. https://github.com/nervosnetwork/cfn-node/pull/96/commits/11f710bafc809af779441daf1eb73b1b603e185c. TLDR is that I moved the code to `on_channel_ready` function. It has to be delayed to there because incrementing these numbers too soon may cause problems.